deps/hashicorp/docker-compose.yml

```diff
@@ -23,7 +24,10 @@ services:
     <<: *container-common
     build: .
     environment:
-      VAULT_ADDR: ${VAULT_ADDR-http://hashicorp:8200}
+      VAULT_ADDR: ${VAULT_ADDR-https://hashicorp:8200}
```

Comment:
I would rather not make TLS the default dev setup since it introduces another area of potential additional on our daily work instead I would rather have `vault-tls` service and everything in place in decide whether or not to use TLS only when we intend to test QKM with it